cumulusci/cli/cci.py

```diff
@@ -764,7 +764,10 @@ def service_info(config, service_name, plain):
 @click.argument("org_name", required=False)
 @pass_config
 def org_browser(config, org_name):
-    org_name, org_config = config.get_org(org_name)
+    try:
```

Comment:
I'll defer to @davisagli on this, but I think it makes sense to apply _Extract Method_ to this try..catch to keep it DRY (Don't Repeat Yourself).

Reply:
@jstvz @davisagli (Sorry for the delay in getting back to this...)

It's hard to tell from the conversations if I should do anything here or not. One thing that I could see being beneficial is moving `raise click.ClickException(f"Org {org_name} does not exist in the keychain")` into its own method so that the error message only exists in a single place.